src/Parallel/traversals/greedy_color.jl

```diff
@@ -1,10 +1,10 @@
 function random_greedy_color(g::AbstractGraph{T}, reps::Integer) where {T <: Integer}
-    best = @distributed (LightGraphs.best_color) for i in 1:reps
+    best = @distributed (LightGraphs.Traversals.best_color) for i in 1:reps
         seq = shuffle(vertices(g))
         LightGraphs.perm_greedy_color(g, seq)
     end
 
-    return convert(LightGraphs.Coloring{T}, best)
+    return convert(LightGraphs.Traversals.Coloring{T}, best)
```

Comment:
Not sure I would put coloring in Traversals, it doesn't feel like one to me

Reply:
It's probably not, but we'll have a chance to move it once we modularize the rest. The interim PRs against 2.0.0 aren't intended to be "here's a version you should use now", but they are intended to be "here's a version that behaves like the latest release". The colorings are available via `using LightGraphs`; we'll eventually change the depwarn to point to a more appropriate place, but for now there are probably 20-30 more PRs that need to be submitted before things can shuffle around.

Reply:
but it looks like a breaking change if you are using LightGraphs in a qualified way:
```
LightGraphs.Coloring -> LightGraphs.Traversals.Coloring
```

Reply:
It's not breaking because the old `greedy_color` is being exported:
```
julia> using LightGraphs
[ Info: Precompiling LightGraphs [093fc24a-ae57-5d10-9952-331d41423f4d]
julia> greedy_color
greedy_color (generic function with 1 method)
julia> using LightGraphs.Parallel
julia> Parallel.greedy_color
greedy_color (generic function with 1 method)
```

This is the same as the existing behavior.

Reply:
ok got it